.changeset/tame-ducks-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/adapter-prisma-legacy': major
+---
+
+Changed default migrationMode from `dev` to `prototype`
```

Comment:
This is because of cypress tests (we could technically change the migrationMode in the test projects because but meh, no one should be using the PrismaAdapter directly anyway so the default doesn't really matter)